### PR TITLE
Feature/global changes

### PR DESCRIPTION
**Pull request by d0whc3r**

### CODE REVIEW

**Comment by Panthro** [comment]
no coverate, no dice... :-)

File: package.json [hunk 1]
```diff
@@ -84,7 +102,12 @@
     ],
     "moduleNameMapper": {
       "^react-native$": "react-native-web"
-    }
+    },
+    "modulePaths": [
```
Comment (Panthro) [inline package.json]: Are you sure this won't hurt more than help?
I really like to have full IDE support....

File: src/router/index.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Route, Router, IndexRoute, hashHistory } from 'react-router';
+
+import App from '../components/App/App';
+
+export default (
+  <Router history={hashHistory}>
```
Comment (Panthro) [inline src/router/index.js]: we cannot use the hash...we need full url paths, with browser support to be able to render well on serve.

**Review by Panthro — CHANGES_REQUESTED** [review]

**Review by Panthro — APPROVED** [review]

**Comment by d0whc3r** [comment]
just restart the job

**Comment by d0whc3r** [comment]
rety job in travis will work fine

**Pull request merged**